jobs/config.json

```diff
@@ -2292,6 +2292,24 @@
       "UNKNOWN"
     ]
   },
+  "ci-kubernetes-e2e-gce-gci-qa-m62": {
+    "args": [
+      "--check-leaked-resources",
+      "--env-file=jobs/platform/gce.env",
+      "--env-file=jobs/env/ci-kubernetes-e2e-gce-gci-qa-m62.env",
```

Comment:
oh - meh - remove the --env-file flags :-)

Reply:
yeah, done. realized it soon after.